Framework/Algorithms/test/ExtractSpectraTest.h

```diff
@@ -158,6 +158,22 @@ class ExtractSpectraTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(ws->x(4)[0], 3.0);
   }
 
+  void test_x_data_is_not_coppied() {
+    Parameters params;
+    params.XMin = 1;
+    params.XMax = 5;
+
+    const auto ws = runAlgorithm(params);
+
+    if (!ws)
+      return;
```

Comment:
should this not raise a failure?

Reply:
Good point, I think I was just copying other tests, but this is certainly somewhere where bugs could get hidden 👍